src/Components/index.js

```diff
@@ -6,4 +6,5 @@ import { NoteInput } from "./NoteInput/NoteInput";
 import { NoteCard } from "./NoteCard/NoteCard";
 import { EditNote } from "./EditNote/EditNote";
 import { ArchivedList } from "./ArchivedList/ArchivedList";
-export { Navbar, Sidebar, ToastWrapper, NotesList, NoteCard, NoteInput, EditNote, ArchivedList };
+import { TrashList } from "./TrashList/TrashList";
+export { Navbar, Sidebar, ToastWrapper, NotesList, NoteCard, NoteInput, EditNote, ArchivedList, TrashList };
```

Comment:
suggestion: you can directly export the components, instead of importing and then exporting